### ISSUE TITLE
Make Login instruction text configurable

### ISSUE THREAD

**Issue by creativeanalyst** [globaleaks/GlobaLeaks#2963]
Checking if it is possible to have this login instruction text configurable.

Have you already filed a report? Enter your receipt.

**Comment by evilaliv3** [comment]
Thank you for this question @creativeanalyst 

Actually we consider this text part of the application as it deals with important aspects (the login) and that summarizes around the application workflow (relation between report and receipt that are key words in the application and for this reason we have not implemented the possibility to customize it.

Which is your current need?

Eventually with this considerations in mind you could use the generic translation override function that you find in the administrative panel within the site settings.

**Comment by creativeanalyst** [comment]
Thinking about having generalized text focused on word 'Submission' instead 'Filing'. We are trying to explore use case of the platform where 'Filing' word may not go well.

How about hardcoding general text like 'Enter receipt number to view previously submitted request'

**Comment by creativeanalyst** [comment]
or Login using receipt number of previously submitted request

**Comment by evilaliv3** [comment]
Actually the current text is the result of many user interviews and research. submission is the text that was previsouly used and that is not replaced by report that is more specific.

**Comment by creativeanalyst** [comment]
Agree. Current text is more appropriate in the context of whistleblowing requests but I was suggesting it to generalize for other use cases of the platform.
